### PR TITLE
Pool-1101 Minor Matic Updates & Surfacing more info

### PR DESCRIPTION
**Pull request by ddesrosier**
- Link to instructions for connecting to L2 (commented out until we have a link)
- Hardcoded Matic prize pool list
- Link to Zapper for token bridging
- Update UI to surface more information on depositor and stats pages (screenshots)
   - Added token names to be more visible (not just symbols)
   - Sponsorship deposits
   - Pool Owner
   - Reserve & reserve rate
   - Token faucet daily drip rates
 
Pool Stats - full list
Screen Shot 2021-03-30 at 3 26 12 PM

Depositor View - short list & token balances & odds
Screen Shot 2021-03-30 at 3 04 10 PM

### CODE REVIEW

File: lib/components/BlockExplorerLink.jsx [hunk 1]
```diff
@@ -20,33 +21,36 @@ export const BlockExplorerLink = (props) => {
...
+                'sm:inline-block': !shorten
+              })}
+            >
+              {display}
+            </span>
+            <LinkIcon className={iconClassName} />
+          </div>
+        )}
+      </a>
+      {copyable && <CopyIcon className='ml-2 my-auto' text={display} />}
```
Comment (chuckbergeron) [inline lib/components/BlockExplorerLink.jsx:52]: I like this 👍

File: lib/components/IndexContent.jsx [hunk 2]
```diff
@@ -105,10 +96,10 @@ const UnsupportedNetworkCard = () => {
 
   return (
     <div className='text-left mb-10 border-2 border-primary rounded-lg px-7 py-4 text-accent-1'>
-      <CardTitle noMargin>☹️ Index Unavailable for {networkName}</CardTitle>
+      <CardTitle noMargin>☹️ Full Index Unavailable for {networkName}</CardTitle>
       <p>
-        Unfortunately due to limitations of Matic we can't dynamically compile a list of created
```
Comment (chuckbergeron) [inline lib/components/IndexContent.jsx:110]: good catch

File: lib/components/PoolStats.jsx [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+const CreditMaturationDaysStat = (props) => (
+  <Stat
+    title='Exit fee decay time'
+    value={`${props.creditMaturationInDays} day${props.creditMaturationInDays === 1 ? '' : 's'}`}
+  />
+)
+
+// APR Stats
+
+// TODO: Need to fetch data & token faucet address
```
Comment (chuckbergeron) [inline lib/components/PoolStats.jsx:335]: So I'm guessing this doesn't render currently? Or maybe we just need to remove the TODO ...

**Review by chuckbergeron — CHANGES_REQUESTED** [review]

File: lib/components/PoolStats.jsx (hunk 3 above)
Comment (ddesrosier) [reply lib/components/PoolStats.jsx:335]: Ah that can be removed

**Pull request merged**